config/environments/production.rb

```diff
@@ -1,4 +1,6 @@
 # License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later
+require "active_support/core_ext/integer/time"
```

Comment:
This feels out of place. we have `core_exts.rb` initializer... Thoughts?

Reply:
That's actually in the default generated code from Rails itself.